### ISSUE TITLE
Rebuild from ~scratch

### ISSUE THREAD

**Issue by colindean** [pittmesh/pittmesh.net#164]
We're using Jekyll 3.7.2 right now with 3.8.3 current and 4.0 not far off. I built this site [originally on 1.4.2](https://github.com/pittmesh/pittmesh.net/commit/962e9e568683daa1de05e7aac1401f583997fdb1#diff-e79a60dc6b85309ae70a6ea8261eaf95R35)! Jekyll's changed a lot and some of the stuff that exists no longer exists or is built-in now. 

The site is really small enough that I could rebuild it entirely probably within about 8 hours in something else, including the latest Jekyll.

As a part of this, I would:

- [x] Move to new hosting, getting #97 (Move to HTTPS) for free
- [x] Use GeoJSON as the backend for the map, getting #136 (use GeoJSON) and potentially #142 (export to ArcGIS JSON)
  - [x] Backing with GeoJSON would make #3 possible since it could load from any URL
  - [x] Backing with GeoJSON would make #14 and #152 simply an implementation detail of how we build the GeoJSON and colorize it in CSS!
- [x] Use Leaflet.js instead of ModestMaps, getting #163 out
  - [x] This makes #1 also doable
- [x] We'll do #165 (MM server deployment) as a finishing step

**Comment by colindean** [comment]
I spent a few hours this weekend investigating using [Vuepress](https://vuepress.vuejs.org/), mostly because I've wanted to learn a little more about JavaScript front-end frameworks. I've decided that it's not ready for the simple level of customization that I want to do: it seems very opinionated thus far and it doesn't actually produce rendered HTML as it builds the page almost entirely in JavaScript at runtime. I tried to use a Leaflet component for Vue to do the map – it had the promise of quick integration – but I couldn't get it to render the map correctly. It was likely my fault but with the realization that Vuepress doesn't output the HTML to disk, it's not worth pursuing further.

So, it's either Jekyll or Hugo. 

I'm kinda leaning Jekyll because I'll be able to re-use a good bit of the plugin code I've already developed, including how we currently produce GeoJSON from my janky nodes and links lists. 

Hugo is probably overkill because it's kinda oriented toward sites that have hundreds of pages and I'd have to do more build system work than I really want to do!
